modules/custom/az_alphabetical_listing/config/install/views.view.az_alphabetical_listing.yml

```
@@ -339,19 +337,18 @@ display:
       tags:
         - 'config:field.storage.node.field_az_alphabetical_index'
   az_alphabetical_listing_main:
-    display_plugin: block
     id: az_alphabetical_listing_main
     display_title: 'Alphabetical Listing'
+    display_plugin: block
     position: 1
     display_options:
-      display_extenders: {  }
       display_description: 'Default display for content using the alphabetical listing field.'
+      display_extenders: {  }
     cache_metadata:
       max-age: -1
       contexts:
         - 'languages:language_content'
         - 'languages:language_interface'
         - 'user.node_grants:view'
         - user.permissions
-      tags:
-        - 'config:field.storage.node.field_az_alphabetical_index'
+      tags: {  }
```

It looks like there was a second instance where the tag was being removed:

```suggestion
      tags:
        - 'config:field.storage.node.field_az_alphabetical_index'
```